docs/howto/getting_started_markdown.md

```diff
@@ -0,0 +1,173 @@
+Workspace](https://extensibility.pages.git.alteryx.com/ayx-sdks/getting_started.html#glossary)
+with default files ready to go.
+
+Use this command to create the workspace:
+`ayx_plugin_cli sdk-workspace-init`
+
+This initializes a workspace to manage all of your custom tools and
+plugins.
+
+To create a new tool, use the `create-ayx-plugin` command.
+
+The `create-ayx-plugin` command takes these parameters:
+
+ -   `--tool-name`: (Required) The name of the new Plugin tool
+ -   `--tool-type`: The type of tool to create.
+
+For this example we\'ll name our new tool `MyFirstTool`. We\'ll use the
+tool-type `single-input-single-output` parameter to create the new tool
+with default input and output anchors.
+
+Use this command:
+
+`ayx_plugin_cli create-ayx-plugin`
+
+### Workspace Configuration
+
+When a workspace is created, a `ayx_workspace.json` file is generated
+inside of your workspace directory. This file tracks metadata related to
+your workspace. By default, you do not need to worry about this file
+because it is managed and updated by the CLI commands.
+
+### Create Your Production Installer
+
+After you have implemented and tested your new Ayx Plugin Tool, you
+might want to distribute it to others. In order to do this, you must
+create a YXI installer. The Core SDK CLI provides a command for
+packaging your workspace and tools into an installer.
+
+Run the `create-yxi` command to generate a YXI installer that is ready
+for distribution.
+
+By default, the name of the YXI produced is based on the Ayx Plugin
+Workspace directory name, but you can override the default with the
+`--yxi-name` flag. Alternatively, you can modify the `yxi_name` field in
+the `ayx_workspace.json` file. (See the [Workspace
+Configurations](https://extensibility.pages.git.alteryx.com/ayx-sdks/python-sdk/getting_started.html#workspace-configurations)
+section for details).
+
+### Build the Ayx Plugin Tool
+
+The [sdk-workspace-init]{.title-ref} command sets up the folder
+structure with a `backends` folder, `ui` folder, and `configuration`
+folder. These folders have all the files necessary for Alteryx Designer
+to integrate with your plugin.
+
+Next we\'re going to build your new Ayx Plugin Tool into Designer, where
+you can drag it onto the canvas and connect it to other Designer tools.
+To build `MyFirstTool` into the Designer application, use the
+`designer-install` command.
+
+Use this command to build MyFirstTool into Designer:
+
+`ayx_plugin_cli designer-install`
+
+**Congratulations!**
+
+You just created your first Alteryx Plugin Tool! You can now open
+Alteryx Designer and find your new tool in the Tool Palette.
+
+### Ayx Plugin Tool Execution in Designer
+
+When Designer runs a tool, it must look for an engine to use. In the
+case of our MyFirstTool Python tool, the engine is itself the *Python
+interpreter*. The interpreter is built out of Anaconda and includes all
+of the packages indicated in `requirements.txt`.
```

Comment:
TW review:

When Designer runs a tool, it must look for an engine to use. In the
case of the MyFirstTool Python tool, the engine is itself the *Python
interpreter*. The interpreter is built out of Anaconda and includes all
of the packages indicated in `requirements.txt`.